### PR TITLE
Feat/token list migration

### PR DESCRIPTION
**Pull request by vic-en**
**Before submitting this PR, please make sure**:

- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:
Copies token lists over from template directory to the conf directory.
PR for https://github.com/hummingbot/gateway/issues/148

**Tests performed by the developer**:
- Manual test

**Tips for QA testing**:
- Run usual yarn commands to setup gateway.
 - At `yarn build` the token files should be copied over.

### CODE REVIEW

File: gateway-setup.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ copy_lists () {
   # Make destination folder if needed
   mkdir $HOST_CONF_PATH/lists
   # Copy all files in the source folder to the destination folder
-  cp $TEMPLATE_DIR/lists/*.json $HOST_CONF_PATH/lists
+  cp -rf $TEMPLATE_DIR/lists $HOST_CONF_PATH
```
Comment (vic-en) [inline gateway-setup.sh:44]:
```suggestion
  cp -rf $TEMPLATE_DIR/ $HOST_CONF_PATH
```

**Comment by rapcmia** [comment]
PR update:
- Test latest commit `5631e235`
- Test on local mbp(m1) and Ubuntu22.04 server
- Test this PR and latest development client branch 

Run `yarn && yarn build` and observed:
image
- This does not seem to copy anything to the `conf/list` even we change the permission `sudo chmod 777 conf/list` and re-run `yarn build` for both local and cloud 

---

Test on local mbp(m1)
image
- Observed that it was only able to copy these `lists/*.json files` from `templates/lists` when we ran `gateway-setup.sh`

---

Test on Ubuntu22.04 cloud server
- We recorded the steps here on the loom video where it does not copy *.json files to conf/lists folder for both `yarn build` and `gateway-setup.sh` script
- https://www.loom.com/share/26f531b67c2c4908be8f36ac25b5174c

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "dev:debug": "NODE_ENV=dev DEBUG=* nodemon src/index.ts",
     "start": "/bin/bash ./startup.sh",
     "copy-files": "copyfiles -a 'src/**/schema/*.json' 'src/templates/*.yml' 'src/templates/lists/*.json' 'test/services/data/**/*.*' dist",
+    "copy-files:token-list": "mkdir -p conf/lists && copyfiles -u 'src/templates/lists/*.json' conf/lists",
```
Comment (fengtality) [inline package.json:19]: I think this works if you change `-u` to `-f` here

**Review by fengtality — CHANGES_REQUESTED** [review]
See other PR

**Pull request closed without merging**